### PR TITLE
[5.0] newsfeeds remove deprecated strings

### PR DESCRIPTION
**Pull request by brianteeman**
Removes strings deprecated in #40493

added the list to to the docs manual

### CODE REVIEW

**Comment by laoneo** [comment]
I know we do not have a deprecation policy for strings. But deprecations are here to give developers time to adapt their code. If we deprecate something in 4.4 and remove it in 5.0, then it doesn't make sense as both are released at the same time. So I would target this pr for 6.0.

**Comment by brianteeman** [comment]
in which case then you might as well close this and related pr

**Comment by laoneo** [comment]
The pr's itself are fine, just want to point out that it is hard to understand for 3rd party extension devs, when it gets basically removed at the same time when deprecated.

**Comment by HLeithner** [comment]
I'm fine with deleting the strings in 5.0, I see no problem with deleting language string in a component which actually doesn't have any b/c coverage, means you still have 2 years for migration. in opposite we couldn't do any deprecation in 4.4 because the also get removed in 2 years.

I would suggest to wait for the planed b/c policy meeting (if it comes soon) and decide then.

**Comment by brianteeman** [comment]
FYI I would have just deleted them without even marking them as deprecated as they are component strings that have not been used for a long long time

**Comment by laoneo** [comment]
I have more a problem with the short deprecation timeframe and not with the removal itself.

**Comment by HLeithner** [comment]
2 years is too short?

**Comment by laoneo** [comment]
I'm talking about the release time of 4.4 and 5.0 and not about how long is 4.4 supported.

**Comment by brianteeman** [comment]
@laoneo if this was code and that code was being used then I would agree 100%. If this was in a library or generic language file then I would agree 100%. But this is in a component. 

Also marking a string as deprecated is a pointless thing to do anyway (even though I have done it as requested) as unlike a deprecation note in code it is never logged or displayed.

Finally some of the extra strings that we have might have been unused for ten plus years. A Previous maintainer refused to allow a string to be removed, ever, and no one thought to compromise with him by marking them as deprecated

File: administrator/language/en-GB/com_newsfeeds.ini [hunk 1]
```diff
@@ -81,19 +81,13 @@ COM_NEWSFEEDS_N_QUICKICON_1="News Feed"
 COM_NEWSFEEDS_N_QUICKICON_SRONLY="News Feeds: %d news feeds are published."
 COM_NEWSFEEDS_N_QUICKICON_SRONLY_1="News Feeds: One news feed is published."
 COM_NEWSFEEDS_NEW_NEWSFEED="New News Feed"
-COM_NEWSFEEDS_NEWSFEEDS="News Feeds"
```
Comment (heelc29) [inline administrator/language/en-GB/com_newsfeeds.ini:84]: please check #40533

File: administrator/language/en-GB/com_newsfeeds.ini [hunk 2]
```diff
@@ -81,19 +81,13 @@ COM_NEWSFEEDS_N_QUICKICON_1="News Feed"
...
 COM_NEWSFEEDS_NO_ITEM_SELECTED="No news feeds selected."
 COM_NEWSFEEDS_NONE="None"
 COM_NEWSFEEDS_NUM_ARTICLES_HEADING="# Articles"
 COM_NEWSFEEDS_NUM_ARTICLES_HEADING_ASC="# Articles ascending"
 COM_NEWSFEEDS_NUM_ARTICLES_HEADING_DESC="# Articles descending"
 COM_NEWSFEEDS_RIGHT="Right"
-COM_NEWSFEEDS_SAVE_SUCCESS="News feed saved."
```
Comment (heelc29) [inline administrator/language/en-GB/com_newsfeeds.ini:91]: please check #40533

File: administrator/language/en-GB/com_newsfeeds.ini (hunk 1 above)
Comment (heelc29) [inline administrator/language/en-GB/com_newsfeeds.ini:84]: please check #40493

**Comment by heelc29** [comment]
@brianteeman Can you please update this PR?
image

**Pull request closed without merging**